Snowflake CCM Object Tags Docs (#27464)
DataDog/documentation: content/en/cloud_cost_management/saas_costs.md

Your Snowflake cost data for the past 15 months can be accessed in Cloud Cost Management after 24 hours. To access the available data collected by each SaaS Cost Integration, see the [Data Collected section](#data-collected).

> **Snowflake CCM Object Tags**

Reviewer: ```suggestion
**Snowflake CCM object tags**
```

> Object tags are user-defined strings that you can attach to Snowflake objects for enhanced auditability and cost analysis. For example, if you want to track costs by team, tag your warehouses with the teams that use them.

Reviewer: ```suggestion
Object tags are user-defined strings that you can attach to Snowflake objects for enhanced auditability and cost analysis. For example, to track costs by team, tag your warehouses with the respective teams that use them.
```

All object tag configuration is done within [Snowflake][105].

> Notes:
> - **Tag Inheritance**: Snowflake objects follow a hierarchy; the CCM integration accounts for inherited tags when submitting cost data.

Reviewer: ```suggestion
Notes:
- **Tag Inheritance**: Snowflake objects adhere to a hierarchical structure, and the CCM integration considers inherited tags when submitting cost data.
```
